#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
uint8 num_obstacle
uint8 timestamp
uint8 application_version
uint8 active_version_number_section
bool left_close_rang_cut_in
bool right_close_rang_cut_in
uint8 go
uint8 protocol_version
bool close_car
uint8 failsafe
uint8 reserved10